src/main/java/io/emeraldpay/dshackle/upstream/ethereum/json/TransactionJsonSnapshot.java

```diff
@@ -0,0 +1,37 @@
+package io.emeraldpay.dshackle.upstream.ethereum.json;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.emeraldpay.etherjar.domain.BlockHash;
+import io.emeraldpay.etherjar.domain.TransactionRef;
+import io.emeraldpay.etherjar.rpc.json.TransactionRefJson;
+
+import java.io.Serializable;
+
+@JsonDeserialize(using = TransactionJsonSnapshotDeserializer.class)
+public class TransactionJsonSnapshot extends TransactionRefJson implements TransactionRef, Serializable {
```

Comment:
Why java?

Reply:
just like other objects and serializers/deserializers are under java package